pysqldb3/geopackage.py

```diff
@@ -499,84 +499,87 @@ def read_gpkg(self, precision=False, private=True, gpkg_encoding=None, print_cmd
                 cmd_env['PGCLIENTENCODING'] = 'UTF8'
 
             if print_cmd:
-                print(print_cmd_string([self.dbo.password], cmd))
+                print(print_cmd_string([dbo.password], cmd))
 
             try:
                 ogr_response = subprocess.check_output(shlex.split(cmd), stderr=subprocess.STDOUT, env=cmd_env)
                 print(ogr_response)
             except subprocess.CalledProcessError as e:
                 print("Ogr2ogr Output:\n", e.output)
                 print('Ogr2ogr command failed. The Geopackage was not read in.')
-                raise subprocess.CalledProcessError(cmd=print_cmd_string([self.dbo.password], cmd), returncode=1)
+                raise subprocess.CalledProcessError(cmd=print_cmd_string([dbo.password], cmd), returncode=1)
 
-            if self.dbo.type == 'PG':
-                self.dbo.query(FEATURE_COMMENT_QUERY.format(
-                    s=self.schema,
+            if dbo.type == 'PG':
+                dbo.query(FEATURE_COMMENT_QUERY.format(
+                    s=schema,
                     t=output_name,
-                    u=self.dbo.user,
+                    u=dbo.user,
                     p=self.path,
                     gpkg=self.gpkg_name,
                     d=datetime.datetime.now().strftime('%Y-%m-%d %H:%M')
                 ), timeme=False, internal=True)
 
             if not private:
                 try:
-                    self.dbo.query('grant select on {s}."{t}" to public;'.format(
-                        s=self.schema, t=output_name),
+                    dbo.query('grant select on {s}."{t}" to public;'.format(
+                        s=schema, t=output_name),
                         timeme=False, internal=True, strict=True)
                 except:
                     pass
-            self.rename_geom()
+            self.rename_geom(dbo, schema, table)
 
-    def rename_geom(self):
+    def rename_geom(self, dbo = None, schema = None, table = None, port=5432):
```

Comment:
can you just call the shapefile.rename_geom? or move that to util or something so they can both access it

Reply:
I can't call the same `rename_geom` function because the `Shapefile` one only takes `self` as an argument while my gpkg version has `dbo`, `schema` and `table` arguments, since I moved these inputs out of the class. Would require me to add more arguments to the Shapefile.py file, which I can do if you want

Reply:
ok I added this as an issue to fix https://github.com/safety-analytics-mapping/pysqldb3/issues/89